src/products/android-l10n/adding_projects.md

```diff
@@ -1,21 +1,27 @@
 
-The TOML file should be pre-populated with a list of locales. When a new project is announced, the mobile PM typically reaches out to localizers on existing communication channels, so they can opt in their locale. If the Android project shares strings with `android-components`, then you need to first make sure the locale code is already listed under its corresponding [TOML file](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/l10n.toml). If not, it will need to be added there as well.
+The TOML file should be pre-populated with a list of locales. When a new project is announced, you typically reach out to localizers on existing communication channels, so they can opt in their locale. If the Android project shares strings with `android-components`, then you need to first make sure the locale code is already listed under its corresponding [TOML file](https://hg.mozilla.org/mozilla-central/file/tip/mobile/android/android-components/l10n.toml). If not, it will need to be added there as well.
 
-The `firefox-android` repository currently uses a [single workflow](https://github.com/mozilla-mobile/firefox-android/blob/main/.github/workflows/import-l10n.yml) (GitHub action) to import strings for all projects from `android-l10n`. Sync with mobile developers to make sure that the code to import strings for this project has been added.
+Localized files are stored in GitHub under the `android-l10n` repository [here](https://github.com/mozilla-l10n/android-l10n). A GitHub [workflow](https://github.com/mozilla-l10n/android-l10n/tree/main/.github/workflows) is used to automatically extract source strings and TOML files from [`gecko-dev`](https://github.com/mozilla/gecko-dev) (the GitHub mirror of `mozilla-central`) which ensures the latest strings land in the `android-l10n` repository.
 
-On the `android-l10n` repository side, you will need to set up [automation](https://github.com/mozilla-l10n/android-l10n/tree/main/.github/workflows) to import strings from the code repository. This automation will be responsible for creating all needed files in the repository. You may need to sync up with a technical localization EPM to set this up.
+TaskCluster is used to import translations and TOML files to [`autoland`](https://hg.mozilla.org/integration/autoland/), then code sheriffs merge new translations to `mozilla-central` together with other code changes. This allows sheriffs to back out updates with errors that break Android builds. TaskCluster is also used to uplift strings from `mozilla-central` to `mozilla-beta`. Sync with release engineers to make sure that the automation to import strings for this project has been added.
```

Comment:
Not blocking, but as a follow-up we should find and add some information here: 
* How often are sheriffs merging code from autoland and at what time.
* How often is the taskcluster automation running to import to autoland, to uplift to beta.